examples/zero-nextjs/src/app/layout.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import type { Metadata } from 'next';
+import '@mui/zero-runtime/styles.css';
```

Comment:
Is this mentioned in the zero runtime README? If not, this needs to be mentioned in the docs that this file needs to be imported in the entry point of the Application.

Reply:
Nice catch.

Reply:
Updated the readme.